taxcalc/utils.py

```diff
@@ -4,10 +4,10 @@
 from collections import defaultdict
 
 STATS_COLUMNS = ['c00100', '_standard', 'c04470', 'c04600', 'c04800', 'c05200',
-                 'c62100','c09600', 'c05800', 'c09200', '_refund', 'c07100',
-                 '_ospctax','s006']
+                 'c62100','c09600', 'c05800', 'c09200', '_refund', 'c07100', '_ospctax',
+                 's006']
 
-TABLE_COLUMNS = ['s006','c00100', 'num_returns_StandardDed', '_standard',
```

Comment:
This a merge conflict where I deleted my version and keep the ospc master version. Not sure why it shows the diff like this.